ToC and Prelim. Script for Tutorials 10, 11 and 12 (#161)
w3c/wot-cg: Tutorials/whatiswot/11-Applications in Smart Homes and Smart Cities/README.md

# Applications in Smart Homes and Smart Cities

## Table of Contents

- Smart Homes
- Architecture Patterns
  - Controllers
  - Gateways
- Smart Cities

## Preliminary Script

> - In this two-part tutorial, we will introduce the W3C Web of Things by exploring its application domains and common architecture patterns. In this video our focus will be Smart Homes and Smart Cities.

Reviewer: ```suggestion
- In this two-part tutorial, we will introduce the W3C Web of Things by exploring its application domains and common architecture patterns. In this video, our focus will be Smart Homes and Smart Cities.
```

### Smart Homes

- In smart homes, the ability to connect and manage multiple assets remotely over the network is highly beneficial.

> - The interesting part about smart homes isn’t tied to a specific solution you buy, but rather in how WoT enables flexibility. Manufacturers can provide Thing Descriptions (TDs) for their devices, which lets anyone build and customize their own systems. By following these open standards, devices can communicate with each other across brands, allowing for seamless integration. This approach ensures that devices remain adaptable and private, and don’t become obsolete, as they can continuously support new use cases and technologies.

Reviewer: ```suggestion
- The interesting aspect of smart homes isn’t tied to a specific solution you buy, but rather in how WoT enables flexibility. Manufacturers can provide Thing Descriptions (TDs) for their devices, which lets anyone build and customize their own systems. By following these open standards, devices can communicate with each other across brands, allowing for seamless integration. This approach ensures that devices remain adaptable and private, and don’t become obsolete, as they can continuously support new use cases and technologies.
```

> - This helps within various activities within the home, such as controlling lights, adjusting room temperature, managing home appliances, and enhancing security using smartphones. These capabilities enable optimization of energy and resource consumption.

Reviewer: ```suggestion
- This helps with various activities within the home, such as controlling lights, adjusting room temperature, managing home appliances, and enhancing security using smartphones. These capabilities enable optimization of energy and resource consumption.
```

**Graphic:** `House with many WoT-powered devices, showcasing Thing Description of the house`

#### Architecture patterns

> In smart homes, we can already notice common architecture patterns. In this video, we will explain Controller and Gateway patterns.

Reviewer: ```suggestion
In smart homes, we can see common architecture patterns. In this video, we will explain Controller and Gateway patterns.
```

1. **Controllers:**

> - A common pattern involves using a controller, both locally and remotely, to interact with devices via the home network by consuming the Thing Description.

Reviewer: ```suggestion
- A common pattern involves using a controller, locally and/or remotely, to interact with devices on the home network by consuming the Thing Description.
```

Reviewer 2: We probably need to adjust the voice generator's text.

- Devices are represented by TDs and act as servers, responding to commands from client controllers (e.g., smartphones) for actions such as adjusting settings or activating sensors.

**Graphic:** `TD of the house transitions into a phone interface (controller) demonstrating interactions with different devices`

2. **Gateways:**

- Consumer can control it both locally or fully remotely.
- Gateways connect home networks to the internet, enabling remote management and monitoring.
- They facilitate communication between local devices and external controllers, enhancing accessibility and control.

**Graphic:** `Old Video 11 Cloud Gateway Animation`

### Smart cities

> - Similar to smart homes, monitoring of smart cities is possible. In this case, end users are different from citizens to municipalities and government agencies.

Reviewer: ```suggestion
- Similar to smart homes, smart cities are possible. In this case, end users are akin to citizens of municipalities and government agencies.
```

> - Smart cities need to cover multiple domains like water management / air quality monitoring and etc. (Like a website selling cars/pumps/breaks (multiple things in one place)). Since WoT is composed of open standards, it can be reliably used by municipalities.

Reviewer: ```suggestion
- Smart cities need to cover multiple domains like water management, air quality monitoring, etc., similar to a website selling cars, pumps, brakes — i.e., multiple things in one place. Since WoT is composed of open standards, it can be reliably used by municipalities.
```

> - Smart cities need to cover multiple domains like water management, air quality monitoring, etc., similar to a website selling cars, pumps, brakes — i.e., multiple things in one place. Since WoT is composed of open standards, it can be reliably used by municipalities.

Reviewer 2: It should be more about WoT being a domain-agnostic standards, so it can scale to the size of a city.

> - In smart cities, bridges, dams, and canals can be monitored for material conditions or deterioration for maintenance repair work to prevent significant damage. Parking slots and street lights can be monitored for crowd management and optimized usage too.

Reviewer: ```suggestion
- In smart cities, bridges, dams, and canals can be monitored for material conditions and deterioration calling for maintenance repair work to prevent significant damage. Parking spaces and street lights can be monitored for crowd management and optimized usage, too.
```

> - In smart cities, bridges, dams, and canals can be monitored for material conditions and deterioration calling for maintenance repair work to prevent significant damage. Parking spaces and street lights can be monitored for crowd management and optimized usage, too.

Reviewer 2: Should be changed to reflect the benefit of WoT. Currently, this is just IoT.

Reviewer 2: or it can be removed

> - Different managers who manage different Things - multiownership.
> 
> > Show TDs for every Thing.
> 
> Since it’s so big there’s usually a mix of architectures.

Reviewer: ```suggestion
- Different managers who manage different Things — multi-ownership.

> Show TDs for every Thing.

Since it’s so big, there’s usually a mix of architectures.
```

> Since it’s so big, there’s usually a mix of architectures.

Reviewer 2: Should include that WoT works well with client-server and publish subscribe architectures or with and without the presence of cloud.

2. **Gateways:**

- Gateways are another pattern commonly found in Smart Homes.

- Gateways connect home networks to the internet, enabling remote management and monitoring.

- They can use the local TD of a device and provide web-level access management, allowing devices to be accessed and managed both locally and remotely over the web. Internally, the device uses an app for local control, while externally, a protocol is used to manage remote interactions.

- They facilitate communication between local devices and external controllers, enhancing accessibility and control.

**Graphic:** `Old Video 11 Cloud Gateway Animation`

### Smart cities

- In smart cities, WoT's domain-agnostic nature enables scalability across various municipal systems, making it more flexible than traditional IoT solutions. 

- WoT's ability to seamlessly integrate different domains — such as water management, air quality monitoring, and traffic control — makes it a perfect fit for urban environments. WoT leverages open standards like Thing Descriptions (TDs) and built-in web protocols in addition to traditional IoT protocols and this allows devices from different systems and manufacturers to communicate.

> - This flexibility ensures smart cities can scale as needed, adding new devices and systems without being tied to a single vendor or technology. Multi-ownership also becomes an option - different organisations or managers oversee different Things, while WoT ensures seamless interoperability across all of them.

Reviewer: ```suggestion
- This flexibility ensures smart cities can scale as needed, adding new devices and systems without being tied to a single vendor or technology. Multi-ownership also becomes an option — different organisations or managers oversee different Things, while WoT ensures seamless interoperability across all of them.
```
